app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -339,7 +339,7 @@ export const sortListings = (
         additional.push(listing)
       }
     } else {
-      if (areLotteryResultsShareable(listing)) {
+      if (isLotteryCompleteDeprecated(listing)) {
```

Comment:
We swant to use `isLotteryCompleteDeprecated` but this function handles both the new Salesforce field as well as the deprecated one. Once we are sure that Salesforce has fully transitioned to the new field, we can replace all instances of `isLotteryCompleteDeprecated` with `isLotteryComplete`, which will be tracked in this ticket: https://sfgovdt.jira.com/browse/DAH-1802